pkg/controller/controller.go

```diff
@@ -272,6 +272,7 @@ func (c *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 
 	drive, err := matchDrive()
 	if err != nil {
+		utils.JSONifyAndLog(req.GetAccessibilityRequirements())
```

Comment:
Bringing back this log, as this will be useful to check if the volumes were asked on wrong nodes. (in topology constraint failure cases)

Reply:
I will revert this change and bring these changes as a separate PR which can take in logging improvisations.